docs/config.md

```diff
@@ -201,6 +201,14 @@ version = 2
     [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
       [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
         endpoint = ["https://registry-1.docker.io", ]
+
+  # 'plugins."io.containerd.grpc.v1.cri".image_decryption' contains config related
+  # to the handling decryption of encrypted container images.
+  [plugins."io.containerd.grpc.v1.cri".image_decryption]
+    # key_model sets how the trust model of where keys should reside for the
```

Comment:
s/sets how the trust model of where keys should reside for the/sets the scope of the trust model for/

How's that sound?

Reply:
sounds good!